### DISCUSSION TITLE
Vender minha empresa para empresa no exterior

### DISCUSSION THREAD

**Discussion by bolds07** [DyegoCosta/trabalhando-remoto#204] [General]
Existe uma empresa de Israel oferecendo uma quantia bastante alta para comprar os direitos de exploração de um aplicativo que eu tenho publicado no google play.

Este aplicativo é o principal ativo da minha empresa e de onde vem 95% do meu faturamento, vender a propriedade intelectual seria basicamente como vender a empresa inteira, o volume financeiro remanescente seria inviavel para manter o PJ ativo e seria mais conveniente encerrar

A empresa que me fez a oferta não tem operação no Brasil e nem tem interesse em faze-lo, sendo assim outros ativos e passivos existente no brasil seriam liquidados junto a operação.

Sei que em caso de venda de empresa no brasil, para entidade no Brasil eu declararia como ganho de capital em PF pagaria 15% de imposto e seguiria minha vida... mas como fazer no caso de exterior?

pensei em declarar uma nota fiscal como venda de propriedade intelectual para a empresa mas isso me colocaria pra fora do teto do Simples e a quantidade de imposto paga nisso é surreal

Alguem sabe como operacionalizar isto?
Algum contador BOM querendo cliente para resolver isso?
Alguem ai com contato no panamá ou nas caymans para eu abrir uma conta [risos]

**Comment by fforbeck** [comment]
e aí @bolds07, blz?! Fiquei curioso para saber se deu certo e como vc operacionalizou. Rola compartilhar sua experiência?

**Comment by lucasven** [comment]
sei que a thread já é antiga, mas vou deixar meu comentario aqui pra referencia futura, você precisa de mais que um contador, você precisa de um advogado de direito internacional. Na issue #161 eles mencionam advogados. Liga pros caras e ve se conseguem te ajudar.
